.storybook/webpack.config.js

```diff
@@ -8,15 +8,24 @@ module.exports = {
         use:
           'react-native-web-image-loader?name=[name].[ext]&scalings[@2x]=2&scalings[-3x]=3',
       },
+      {
+        test: /\.js$/,
+        include: [
+          path.join(__dirname, '../node_modules/react-native-multi-slider'),
```

Comment:
I will need to evaluate whether this approach will work for us; I've been trying to reduce the amount of stuff that goes into webpack config files as just installing the library might not work.

Reply:
If you try to install @kiwicom/universal-components@0.0.9-alpha.0 in a web project, it does not work, even after modifying the webpack config to load the files apparently.

https://github.com/kiwicom/margarita/tree/testing-slider-kiwicom-alpha-release